TradeItIosTicketSDK2/TradeItOrder.swift

```diff
@@ -29,6 +29,7 @@ public typealias TradeItPlaceOrderHandlers = (
     @objc public var expiration: TradeItOrderExpiration = TradeItOrderExpirationPresenter.DEFAULT
     @objc public var userDisabledMargin = false
     @objc public var quantity: NSDecimalNumber?
+    public var quantityType: OrderQuantityType?
```

Comment:
Unable to add `@objc` here because `OrderQuantityType` is an `enum: String` that is not compatible. We could look at changing it, but Y! isn't currently using it so not urgent.

Reply:
I'll mark it internal for now